src/EnergyPlus/ElectricPowerServiceManager.hh

```diff
@@ -162,6 +124,8 @@ public: // Methods
 	Real64
 	getACEnergyOut();
 
+	std::string
+	name();
```

Comment:
You can return by const reference to avoid expensive string copy since `name()` is just returning an existing string. Methods that don't modify the object should be declared `const`.
```
std::string const &
name() const;
```
Same comment for all the `name()` methods below.